ui/raidboss/popup-text.js

```diff
@@ -587,11 +588,8 @@ class PopupText {
         }
       }
 
-      if ('soundVolume' in trigger)
-        soundVol = ValueOrFunction(trigger.soundVolume);
-
       soundUrl = triggerOptions.SoundOverride || soundUrl;
-      soundVol = triggerOptions.VolumeOverride || soundVol;
+      soundVol = triggerOptions.VolumeOverride || triggerSoundVol || soundVol;
```

Comment:
Is this from a separate change?

Reply:
Reply for others, it's from the middle commit d5b001c.